### PR TITLE
[TT-561] move chainlink-env as a package

### PR DESCRIPTION
**Pull request by Tofel**

### CODE REVIEW

**Comment by AnieeG** [comment]
is there a way to break this up in multiple PRs?

**Comment by Tofel** [comment]
> is there a way to break this up in multiple PRs?

I'd rather not, although I know it's huge. Would it help if I went through commits and listed all files that were modified instead of copied?

File: .github/workflows/env-publish-test-bash-image.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Publish Test Base Image (Env)
```
Comment (tateexon) [inline .github/workflows/env-publish-test-bash-image.yaml:1]: The filename has a typo, should be ...test-base-image.... not bash

File: env/logging/log.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package logging
```
Comment (tateexon) [inline env/logging/log.go:1]: This file can go away in favor of the logging package already in CTF: https://github.com/smartcontractkit/chainlink-testing-framework/blob/main/logging/log.go
No need to have two separate ones

Comment (Tofel) [reply env/logging/log.go:1]: good catch!

**Comment by Tofel** [comment]
Moved to https://github.com/smartcontractkit/chainlink-testing-framework/pull/747 where changes are better visible

**Pull request closed without merging**